src/AutoRest.CSharp/Common/Output/Models/CmcRestClientBuilder.cs

```diff
@@ -79,16 +79,15 @@ private static string GetRequestParameterName(RequestParameter requestParameter)
         }
 
         /// <summary>
-        /// Build CmcRestClientMethod for mgmt and HLC
+        /// Build CmcRestClientMethod for mgmt only
         /// </summary>
         /// <param name="operation"></param>
         /// <param name="httpRequest"></param>
         /// <param name="requestParameters"></param>
-        /// <param name="responseHeaderModel"></param>
         /// <param name="accessibility"></param>
         /// <param name="returnNullOn404Func"></param>
         /// <returns></returns>
-        public RestClientMethod BuildMethod(Operation operation, HttpRequest httpRequest, IEnumerable<RequestParameter> requestParameters, DataPlaneResponseHeaderGroupType? responseHeaderModel, string accessibility, Func<string?, bool>? returnNullOn404Func = null)
+        public RestClientMethod BuildMethod(Operation operation, HttpRequest httpRequest, IEnumerable<RequestParameter> requestParameters, string accessibility, Func<string?, bool>? returnNullOn404Func = null)
```

Comment:
I removed this parameter because now the only usage of this method comes from mgmt plane, and that removed parameter is always getting null.